Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -482,7 +487,11 @@ open class AVFoundationPlayback: Playback {
         playerLayer = nil
         player?.replaceCurrentItem(with: nil)
         player = nil
+    }
+
+    private func clearObjects() {
```

Comment:
clearObjects do not seem to be the real action performed in this method? It seems to reset Playback properties and so a possible better name to it could be resetPlaybackProperties.

Reply:
Done.